<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindowOccClass</class>
 <widget class="QMainWindow" name="MainWindowOccClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1035</width>
    <height>765</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindowOcc</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="widget_param" native="true">
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0">
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="1">
           <widget class="QSpinBox" name="sb_precision">
            <property name="maximum">
             <number>64</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="sb_row">
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDoubleSpinBox" name="dsb_low">
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>-0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="lb_low">
            <property name="text">
             <string>low</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QDoubleSpinBox" name="dsb_high">
            <property name="minimum">
             <double>-1000.000000000000000</double>
            </property>
            <property name="maximum">
             <double>1000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>5.500000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>row</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>col</string>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="lb_high">
            <property name="text">
             <string>high</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>precision</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="lb_low_2">
            <property name="text">
             <string>distance</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="dsb_distance">
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>3.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="10" column="0" colspan="2">
           <widget class="QPushButton" name="pb_generate">
            <property name="text">
             <string>Regenerate Data</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="sb_col">
            <property name="maximum">
             <number>300</number>
            </property>
            <property name="value">
             <number>7</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="sb_core">
            <property name="maximum">
             <number>64</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>thread</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QPushButton" name="pb_valueMax">
            <property name="text">
             <string>ValueMax</string>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QPushButton" name="pb_valueSmall">
            <property name="text">
             <string>ValueSmall</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <widget class="QWidget" name="gridLayoutWidget">
        <layout class="QGridLayout" name="gridLayout_view"/>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <layout class="QGridLayout" name="gridLayout_Console"/>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1035</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionview"/>
    <addaction name="act_unionfind_ori"/>
    <addaction name="act_unionfind_opt1"/>
    <addaction name="act_unionfind_opt2"/>
    <addaction name="actionkd_find1D"/>
    <addaction name="actionori_find1D"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionFitAll"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionFitAll"/>
   <addaction name="actionview"/>
   <addaction name="actionkd_find1D"/>
   <addaction name="actionori_find1D"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionGenerate">
   <property name="text">
    <string>generate</string>
   </property>
  </action>
  <action name="act_unionfind_ori">
   <property name="text">
    <string>unionfind_ori</string>
   </property>
  </action>
  <action name="actionFitAll">
   <property name="text">
    <string>FitAll</string>
   </property>
  </action>
  <action name="act_unionfind_opt1">
   <property name="text">
    <string>unionfind_opt1</string>
   </property>
  </action>
  <action name="act_unionfind_opt2">
   <property name="text">
    <string>unionfind_opt2</string>
   </property>
  </action>
  <action name="actionview">
   <property name="text">
    <string>view</string>
   </property>
  </action>
  <action name="actionkd_find1D">
   <property name="text">
    <string>kd_find1D</string>
   </property>
  </action>
  <action name="actionori_find1D">
   <property name="text">
    <string>ori_find1D</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
